debian/rules

```diff
@@ -19,16 +19,19 @@ CFLAGS+=$(CPPFLAGS)
 CXXFLAGS+=$(CPPFLAGS)
 
 # Only do a strict symbol checking on Linux
+# https://manpages.debian.org/testing/dpkg-dev/dpkg-gensymbols.1.en.html
+# Level 4: Fails if some libraries have been introduced.
 ifneq (,$(filter linux,$(DEB_HOST_ARCH_OS)))
-    DPKG_GENSYMBOLS_CHECK_LEVEL := 4
+    export DPKG_GENSYMBOLS_CHECK_LEVEL = 4
 endif
 
 BUILDDIR := builddir
 DEB_VERSION_REVISION := $(shell echo $(DEB_VERSION) | sed -e 's/^.*-//')
 DEB_VERSION_VERSION := $(shell echo $(DEB_VERSION) | sed -e 's/^.*:\(.*\)\(-\|+\).*/\1/')
 DEB_VERSION_MAJOR := $(shell echo $(DEB_VERSION_VERSION) | sed -e 's/^\(.*\)\..*$$/\1/')
 RELEASE := $(shell lsb_release -r -s) # Use changelog based DEB_DISTRIBUTION instead?
-TMP:=$(CURDIR)/debian/tmp
+TMP := $(CURDIR)/debian/tmp
```

Comment:
Should TMP be an export? Its not used by rules (aka this makefile)?

Reply:
Probably.. It does not hurt though.

Reply:
> Its not used by rules (aka this makefile)?

Yes it is, `$(TMP)` appears 17 times in `debian/rules`.